os/include/debug.h

```diff
@@ -335,6 +335,22 @@ int get_errno(void);
 #define bmllvdbg(...)
 #endif
 
+#ifdef CONFIG_DEBUG_BINARY_COMPRESSION_ERROR
```

Comment:
1. Let's move this before binmgr.
2. Please add those stubs at line 1096.

Reply:
done.